crates/store/re_chunk_store/src/dataframe.rs

```diff
@@ -793,13 +794,36 @@ impl ChunkStore {
         &self,
         selector: &ComponentColumnSelector,
     ) -> ComponentColumnDescriptor {
+        // Happy path if this string is a valid component
+        let direct_component = ComponentName::from(selector.component_name.clone());
```

Comment:
Hmmm, I guess that means we're going to intern a bunch of random strings indefinitely.

This is hardly a problem in practice, but might deserve a NOTE.

Reply:
Issue: https://github.com/rerun-io/rerun/issues/7699